apps/server/src/positions/entities/position.entity.ts

```diff
@@ -1 +1,41 @@
-export class Position {}
+import { ApiProperty } from '@nestjs/swagger';
+import { Position } from '@prisma/client';
+import { Exclude } from 'class-transformer';
+import { SignatureFieldEntity } from './../../signature-fields/entities/signature-field.entity';
+import { Optional } from '@nestjs/common';
+import { DepartmentEntity } from '@server/departments/entities/department.entity';
+
+export class PositionEntity implements Position {
+  @ApiProperty()
+  id: string;
+
+  @ApiProperty()
+  name: string;
+
+  @ApiProperty()
+  single: boolean;
+
+  @Exclude()
+  createdAt: Date;
+
+  @Exclude()
+  updatedAt: Date;
+
+  @Exclude()
```

Comment:
can we move the department and signature fields above the created/updated at fields?